### ISSUE TITLE
Real scenario, practical sample app with lazy loading in mind 

### ISSUE THREAD

**Issue by un33k** [ngrx/platform#48]
**Feature Request / Document Request**

@ngrx/platform may benefit from a more complete and practical sample app.

This package [ngrx-lazy-load](https://github.com/FabienDehopre/ngrx-lazy-load) by @FabienDehopre is really easy to follow and covers many things.

Lazily loaded modules are only referenced once (indirect), and that is in the root route via the `loadChildren` methods. As per the following example.

```typescript
const routes: Routes = [
  {
    path: '',
    redirectTo: 'home',
    pathMatch: 'full'
  },
  {
    path: 'home',
    component: HomeComponent
  },
  {
    path: 'one',
    loadChildren: './one/one.module#OneModule'
  },
  {
    path: 'two',
    loadChildren: './two/two.module#TwoModule'
  }
];
```

Effects (hacked), Models, Services, Actions, Reducers & States (no Persistence example), are all within the lazily loaded modules themselves without any external direct referencing.

**Comment by Kamshak** [comment]
Yes please! Having quite a bit of trouble figuring this out

**Comment by loiane** [comment]
I've been creating a store directory inside my featureModule with the following files: reducer, actions,  state (with initial state), effects and a service that exposes the store (and retrieving the state with rxjs itself, not with reselect). 
I tried this format with ngrx/platform and it is working ok. Lately, I also tried to package all these files as a sub-module of my feature module as well. 
I've been working on a VsCode plugin to generate the files using this structure of files, and as soon as I have some spare time I'm planning on publishing it and also counting on getting your feedback as well :)
